poc-data-api/Extracting_Water_Temperature_at_Site.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Extracting temperature at specific reefs"
+author: "Denisse Fierro Arcos"
+date: "2023-06-05"
+output: 
+  github_document:
+    toc: true
+    html_preview: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Goal of this notebook
+
+This notebook will demonstrate how to access the RIMReP `geoparquet` collection for AIMS Sea Surface Temperature Monitoring Program. This dataset includes data since 1980 for tropical and subtropical coral reefs around Australia, including 80 Great Barrier Reef sites.

```

Comment:
I think there are more than 80 sites, around 300. Check.

Reply:
This is my bad, I got this information from the [website](https://apps.aims.gov.au/metadata/view/4a12a8c0-c573-11dc-b99b-00008a07204e) describing the dataset and I did not actually check. A quick subset of the data using the maximum spatial extent gives me over 400 points. I have removed the number and rephrased this sentence.